examples/src/main/java/jaxrs/examples/client/builderListener/BasicLoggingClientBuilderListener.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018 IBM and Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jaxrs.examples.client.builderListener;
```

Comment:
According to https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html package names have to be all-lowercase. I would like to ask you to fix this.